packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -73,7 +73,7 @@ export class RouterliciousRestWrapper extends RestWrapper {
                     this.request<T>(config, statusCode)
                         .then(resolve)
                         .catch(reject);
-                }, axiosError.response!.data.retryAfter * 1000));
+                }, axiosError.response!.data.retryAfter));
```

Comment:
do who know that the units are here?

Reply:
Yup, these come in as seconds from the service